src/ensembl/production/datacheck/config.py

```diff
@@ -38,6 +41,29 @@ def get_app_version():
             version = f.read()
     return version
 
+def get_server_names(url, flag=0):
```

Comment:
There is this nice core thing I created for loading remote configs: https://github.com/Ensembl/ensembl-py/blob/d8859c0215b641cb6db4b4e8ac7037ffcc167a76/src/python/ensembl/utils/rloader.py 
I guess we could reuse may be?

Reply:
BTW, the fall back could just be an empty list. So the app start, but empty.

Reply:
we can reuse the RemoteFileLoader